cmd/boulder-ra/main.go

```diff
@@ -31,20 +29,14 @@ func main() {
 
 		go cmd.DebugServer(c.RA.DebugAddr)
 
-		var paDbMap *gorp.DbMap
-		if c.RA.HostnamePolicyFile == "" {
-			dbURL, err := c.PA.DBConfig.URL()
-			cmd.FailOnError(err, "Couldn't load DB URL")
-			paDbMap, err = sa.NewDbMap(dbURL)
-			cmd.FailOnError(err, "Couldn't connect to policy database")
-		}
-		pa, err := policy.New(paDbMap, c.PA.EnforcePolicyWhitelist, c.PA.Challenges)
+		pa, err := policy.New(c.PA.Challenges)
 		cmd.FailOnError(err, "Couldn't create PA")
 
-		if c.RA.HostnamePolicyFile != "" {
-			err = pa.SetHostnamePolicyFile(c.RA.HostnamePolicyFile)
-			cmd.FailOnError(err, "Couldn't load hostname policy file")
+		if c.RA.HostnamePolicyFile == "" {
+			cmd.FailOnError(nil, "HostnamePolicyFile was empty.")
```

Comment:
The language here (and where it is used above) is slightly confusing, perhaps `HostnamePolicyFile must be provided` would be clearer.